### PR TITLE
agda-pkg: init at 0.1.50

### PR DESCRIPTION
**Pull request by alexarice**
###### Motivation for this change

Adds `agda-pkg` program, which we are experimenting with in https://github.com/NixOS/nixpkgs/pull/87903 and may also be useful otherwise to people who do not want to use nix infrastructure for agda

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @turion

### CODE REVIEW

**Review by veprbl — APPROVED** [review]

File: pkgs/development/tools/agda-pkg/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ lib, python3Packages, agda }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "agda-pkg";
+  version = "0.1.49";
```
Comment (jonringer) [inline pkgs/development/tools/agda-pkg/default.nix:7]: newer version in available
```suggestion
  version = "0.1.50";
```

File: pkgs/development/tools/agda-pkg/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  disabled = !(pythonAtLeast "3.6");
```
Comment (jonringer) [inline pkgs/development/tools/agda-pkg/default.nix:9]:
```suggestion
  disabled = pythonOlder "3.6";
```

File: pkgs/development/tools/agda-pkg/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "07smmys9994gncsvd069iicyvj2k3yllwh77p6xr784bk11dnn8q";
+  };
+
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/tools/agda-pkg/default.nix:16]: at least ensure it executes.
```suggestion
  checkPhase = ''
    $out/bin/agda --help > /dev/null
```

**Comment by alexarice** [comment]
Have addressed comments, thanks for the review

**Pull request merged**